addons/docs/package.json

```diff
@@ -137,10 +136,10 @@
     "zone.js": "^0.11.3"
   },
   "peerDependencies": {
-    "@storybook/angular": "6.4.0-alpha.6",
-    "@storybook/vue": "6.4.0-alpha.6",
-    "@storybook/vue3": "6.4.0-alpha.6",
-    "@storybook/web-components": "6.4.0-alpha.6",
+    "@storybook/angular": "*",
```

Comment:
I tried to put `workspace:*` here but it is automatically replaced by `*` later, is it a bad practice? To investigate deeper

Reply:
`peerDependencies` isn't supposed to specify where it comes from, only the allowed range, though in this case perhaps it should be allowed so it can be replaced at publish time

cc @arcanis